e2e/istio.sh

```diff
@@ -17,7 +17,9 @@
 set -eo pipefail
 set -x
 
-HELM="$(pwd)/bin/helm"
+PARENT_DIR="$(cd .. && pwd)"
```

Comment:
We can pass LOCALBIN to the script, like https://github.com/mosn/htnn/blob/e95743bbc1dcb8355619780421ac93a9a787ad1b/Makefile#L44

Reply:
sry, I missed it here.

Please wait for me for a few minutes.